INSTALL.md

```diff
@@ -254,6 +254,8 @@ For example, in a Linux platform, assuming one wishes to execute the `adaboost_d
 ./_cmake_results/intel_intel64_so/adaboost_dense_batch
 ```
 
+DPC++ examples (running on GPU) from oneAPI are also auto-generated within these folders when oneDAL is built with DPC++ support (target `oneapi` in the Makefile), but be aware that it requires a DPC++ compiler such as ICX, and executing the examples requires the DPC++ runtime as well as the GPGPU drivers. The DPC++ examples can be found under `examples/oneapi/dpc`.
```

Comment:
DPC++ examples (running on GPU) - is a little bit incorrect, because it also launches the examples on CPU/other sycl supported devices. Probably sycl supported devices is more accurate here. Other sentences looks good